about.html

```diff
@@ -0,0 +1,201 @@
+            <li><a href=""><img src="shapes/twitter.png" height="20px" height="20px" alt="twitter"></a></li>
+            <li><a href="">My page</a></li>
+            <li><a href="">Logout</a></li>
+          </ul>
+        </div>
+      </div>
+    </div>
+
+    <div class="shapes">
+      <img src="shapes/shape5.png" alt="" class="shape1">
+      <img src="shapes/shape2.png" alt="" class="shape2">
+    </div>
+    <div class="desktop-color">
+      <div class="container">
+        <nav id="navbar">
+          <div class="logo">
+            <a href="index.html" class="logo-link flex">
+              <div class="images flex">
+                <img src="shapes/graphic.png" alt="web development logo">
+                <h1>Dev</h1>
+              </div>
+            </a>
+          </div>
+          <ul class="menu-items flex" id="menu-mobile">
+            <li><a href="about.html">About</a></li>
+            <li><a href="index.html">Contact</a></li>
+            <li><a href="index.html">New</a></li>
+            <li class="signup-btn"><a href="index.html">Signup</a></li>
+          </ul>
+          <div class="btn">
+            <button id="hamburger-btn">
+              <img class="flex" src="shapes/menu1.png" alt="hamburger button">
+            </button>
+            <button id="close-btn">
+              &times
+            </button>
+          </div>
+        </nav>
+      </div>
+
+    </div>
+
+  </header>
+
+  <main>
+    <div class="container">
+      <div class="hero-section flex-column" id="hero-section">
+        <div class="shapes">
+          <img src="shapes/shape7.png" alt="hexagon" class="shape7">
+          <img src="shapes/why-shape-dot.png" alt="shapes dots" class="shape-dot">
+          <img src="shapes/hexagon.png" alt="" class="hexagon">
+        </div>
+        <div class="hero-section-about">
+          <h3 class="hero-first-title-about flex-column">
+            "Become a Pro Web Developer"
+          </h3>
+          <h1 class="hero-title flex-column hero-title-about">
+            Learn Web Development
+          </h1>
+          <div class="hero-desc-about flex-column">
+            <p>
+              Get access to 2700+ web development courses taught by top instructors
+              on our online learning platform. Our courses cover topics for
+              beginners to advanced level.
+            </p>
+            <p>
+              Our top instructors come from leading tech companies
+              and offer expertise in HTML, CSS, JavaScript, and full-stack development.
+              Get started on your web development journey today with our courses.
+            </p>
+          </div>
+        </div>
+        <div class="contactus">
+          <p>
+            Get in touch with us to learn more about our online web development courses.
+            Email us at <a href="#">bahatizirimwabagabo@gmail.com</a> or visit us in Gisenyi city, Rwanda.
+          </p>
+        </div>
+
+      </div>
+    </div>
```

Comment:
Well done on this section 👍🏽 but let's see how the Desktop version can be improved, make sure to increase the font-size (here and in some other places as well), adjust the padding and margin to make it look more like the design. Take a look at the differences👇🏽 

## Figma

![Screenshot 2023-04-28 at 03 30 15](https://user-images.githubusercontent.com/45739963/235083466-4691c25e-e0ca-422a-bbf4-5331d0f98b42.png)

## Your design

![Screenshot 2023-04-28 at 03 31 13](https://user-images.githubusercontent.com/45739963/235083641-e3977856-2ed7-4a42-96ac-b6eaa74edcfb.png)

Reply:
Ok, thank you for the review, i try my best ot fix that one now is fine.